app/__init__.py

```diff
@@ -1,11 +1,43 @@
 import os
 from flask import Flask, render_template, request
 from dotenv import load_dotenv
+import folium
+import json
 
 load_dotenv()
 app = Flask(__name__)
 
 
 @app.route('/')
 def index():
-    return render_template('index.html', title="MLH Fellow", url=os.getenv("URL"))
+    # Create a map object
+    m = folium.Map(min_zoom=2)
+
+    # Get JSON data path
+    locations_file_path = os.path.join(os.path.dirname(__file__), 'static', 'data', 'locations.json')
+
+    # Load member locations from JSON file)
+    with open(locations_file_path, 'r') as f:
+        data = json.load(f)
+        members = data['members']
+        for member in members:
+            color = member['color']
+            locations = member['locations']
+            for location in locations:
+                lat = location['lat']
```

Comment:
Since you're only using each of these variables once, with the exception of location_name, saving them as variables instead of just fetching them from the dictionary when needed is just making the code longer and harder to read. It might make it a little cleaner to take out these middlemen.

Reply:
I can do something like the following to resolve it:
```
marker = folium.Marker(
                    location=[location['lat'], location['lon']],
                    icon=folium.Icon(color=color, icon='star'),
                    popup=folium.Popup(
                        f'''
                        <style>
                            h3 {{font-family: "Roboto", serif; text-align: center;}}
                            p {{font-family: "Roboto", serif; text-align: center;}}
                        </style>
                        <h3>{location['name']}</h3>
                        <p>{location['description']}</p>
                        ''',
                        width=250,
                        height=100
                    ),
                    tooltip=location['name']
                )
```